plugins/ccgx/ccgx.quirk

```diff
@@ -51,7 +51,7 @@ Summary = CCGx Power Delivery Device (Symmetric FW1)
 [DeviceInstanceId=USB\VID_04B4&PID_5218&SID_1F00&APP_6432&MODE_FW2]
 Summary = CCGx Power Delivery Device (Symmetric FW2)
 
-# HP Adicora Dock A Type
+# HP HP USB-C Dock G5
```

Comment:
HP HP ?
Same for the next change

Reply:
I fixed it in new commit below. "ccgx: fix comment in qurik"